doc/userdoc/index.rst

```diff
@@ -7,11 +7,12 @@ Welcome to the NEST simulator documentation!
 
 NEST 3.0 provides a more intuitive experience with simplified yet versatile handling and manipulation of nodes and connections.
 
+You can find all the information in our section :doc:`guides/nest2_to_nest3/index`.
 
-- Visit our :doc:`What's new? <guides/nest2_to_nest3/nest2_to_nest3_overview>` guide to get an overview of NEST 3.0 and the new functionality.
+**Note that some of your simulations may need to be updated to run in NEST 3.0!**
```

Comment:
I think maybe it should be `scripts` instead of `simulations`. Maybe `simulation scripts`.

Reply:
Done!